db/migrate/20200813013953_create_memberships.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMemberships < ActiveRecord::Migration[6.0]
+  def change
+    create_table :memberships do |t|
```

Comment:
Requires uniqueness to prevent multiple memberships between a user and organization.

Reply:
Added uniqueness to prevent duplicate memberships between a user and organization: https://github.com/SpicyCookies/cookie_points_service/pull/2/commits/fd3d301a001bcc345cfdc847c60d787883f99c1c

Reply:
Fixed uniqueness: https://github.com/SpicyCookies/cookie_points_service/pull/2/commits/8a0b2a833ccd3ec81122943bfaaeeca2eb085a9e